<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>conUserItem</class>
 <widget class="QWidget" name="conUserItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>75</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>75</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>75</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>62</width>
       <height>62</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>62</width>
       <height>62</height>
      </size>
     </property>
     <widget class="QLabel" name="icon_lb">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>15</y>
        <width>46</width>
        <height>46</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>46</width>
        <height>46</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>46</width>
        <height>46</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="red_point">
      <property name="geometry">
       <rect>
        <x>27</x>
        <y>2</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="username_lb">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
